patches/components-permissions-permission_request_manager.cc.patch

```diff
@@ -1,15 +1,15 @@
 diff --git a/components/permissions/permission_request_manager.cc b/components/permissions/permission_request_manager.cc
-index 44f5da292099a35a9d8eba4bd8f6e22fa09d3ff5..bca6661405f66583bde15cba6f41ec928b0ce121 100644
+index 44f5da292099a35a9d8eba4bd8f6e22fa09d3ff5..c3d5c551d3c6a51399def26e671b9eb78e283cc6 100644
 --- a/components/permissions/permission_request_manager.cc
 +++ b/components/permissions/permission_request_manager.cc
-@@ -551,6 +551,7 @@ void PermissionRequestManager::OnVisibilityChanged(
-   if (!tab_subscriptions_.empty()) {
-     return;
+@@ -553,6 +553,7 @@ void PermissionRequestManager::OnVisibilityChanged(
    }
-+  BRAVE_PERMISSION_REQUEST_MANAGER_ON_VISIBILITY_CHANGED
    bool prior_tab_is_active_ = tab_is_active_;
    tab_is_active_ = visibility != content::Visibility::HIDDEN;
++  BRAVE_PERMISSION_REQUEST_MANAGER_ON_VISIBILITY_CHANGED
```

Comment:
@simonhong I am not sure if our override is still needed given that the upstream change seems to be fixing the same issue our override does. Could you, please take a look at https://github.com/brave/brave-core/pull/30537/commits/97c9ef1962ee6fdc76347ea9c6455ac5b10f0345 and https://github.com/brave/brave-core/pull/30537/commits/51679aca20d76ef0f9f35ed9f7d4eb897d967d4e.

Reply:
As per https://bravesoftware.slack.com/archives/CU5UXM3EC/p1757401690574529?thread_ts=1757078634.924659&cid=CU5UXM3EC, it would better to use our override (`don't show permission prompt from inactive split tab`) now.
Chromium fixed that issue by activating an inactive split tab when it's about to launch permission dialog.
I had to check how to disable chromium's approach but couldn't find any room with my other issues. could this be handled in f/u?